consensus/transactions.go

```diff
@@ -10,6 +10,7 @@ import (
 // Exported errors
 var (
 	ConflictingTransactionErr = errors.New("conflicting transaction exists in transaction pool")
+	InvalidSignatureErr       = errors.New("invalid signature in transaction")
```

Comment:
I remember writing a bunch of ErrXYZ, I'm not sure when I started doing it this way. It is more readable but it might impact organization and alphabetization.

Except, between grep and chunking things, I'm not really convinced it needs to be a prefix.